datasets/commoncrawl.yaml

```diff
@@ -1,53 +1,67 @@
   - Description: Crawl data (WARC and ARC format)
     ARN: arn:aws:s3:::commoncrawl
     Region: us-east-1
     Type: S3 Bucket
 DataAtWork:
   Tutorials:
-    - Title: Large-scale graph mining with Spark
-      URL: https://towardsdatascience.com/large-scale-graph-mining-with-spark-750995050656
-      AuthorName: Win Suen
-      AuthorURL: https://github.com/wsuen/pygotham2018_graphmining
+    - Title: Analysing Petabytes of Websites
+      URL: http://tech.marksblogg.com/petabytes-of-website-data-spark-emr.html
+      AuthorName: Mark Litwintschik
+      Services:
+        - EMR
+    - Title: Index to WARC Files and URLs in Columnar Format
+      URL: https://commoncrawl.org/2018/03/index-to-warc-files-and-urls-in-columnar-format/
+      AuthorName: Sebastian Nagel
+      Services:
+        - Athena
+    - Title: Common Crawl Index Athena
+      URL: https://skeptric.com/common-crawl-index-athena/
+      AuthorName: Edward Ross
+      Services:
+        - Athena
+    - Title: Search the Common Crawl Using Lambda Functions
+      URL: https://github.com/andresriancho/cc-lambda
+      AuthorName: Andres Riancho
+      Services:
+        - Lambda
+  Tools & Applications:
     - Title: Learning word vectors for 157 languages
-      URL: https://arxiv.org/abs/1802.06893
+      URL: https://www.aclweb.org/anthology/L18-1550
       AuthorName: Facebook AI Research
       AuthorURL: https://fasttext.cc/docs/en/crawl-vectors.html
-  Tools & Applications:
     - Title: Dresden Web Table Corpus (DWTC)
       URL: https://wwwdb.inf.tu-dresden.de/research-projects/dresden-web-table-corpus/
       AuthorName: Database Systems Group Dresden
       AuthorURL: https://wwwdb.inf.tu-dresden.de/
-    - Title: Index to WARC Files and URLs in Columnar Format
-      URL: http://commoncrawl.org/2018/03/index-to-warc-files-and-urls-in-columnar-format/
-      AuthorName: Sebastian Nagel
+    - Title: "CCNet: Extracting high quality monolingual datasets from web crawl data"
+      URL: https://arxiv.org/abs/1911.00359
+      AuthorName: Facebook AI Research
+      AuthorURL: https://github.com/facebookresearch/cc_net
   Publications:
     - Title: Building a Web-Scale Dependency-Parsed Corpus from CommonCrawl
       URL: https://arxiv.org/pdf/1710.01779.pdf
-      AuthorName: Alexander Panchenko, et al.
+      AuthorName: Alexander Panchenko, Eugen Ruppert, Stefano Faralli, Simone Paolo Ponzetto, Chris Biemann
     - Title: Using open data to predict market movements
       URL: https://education.emc.com/content/dam/dell-emc/documents/en-us/2017KS_Ravinder-Using_Open_Data_to_Predict_Market_Movements.pdf
       AuthorName: DELL EMC
     - Title: N-gram counts and language models from the Common Crawl
       URL: http://www.lrec-conf.org/proceedings/lrec2014/pdf/1097_Paper.pdf
       AuthorName: Christian Buck, Kenneth Heafield, Bas van Ooyen
       AuthorURL: http://statmt.org/ngrams/
-    - Title: Large-scale analysis of style injection by relative path overwrite
```

Comment:
Same question here—want to see what the rationale for removing this citation would be

Reply:
If I remember correctly, we added this article back in 2018 as one example for the use case or research topic "internet security". The reason to remove it now is that it has a lower impact compared to the other listed scientific papers. 

It isn't an easy task select a not too big list of data-at-work publications and also to cover various use cases (there's more than NLP). Regarding the two publications: we may keep them - feel free to modify the PR or let me know if I shall include them again.
I think we cannot share here in the Open Data Registry an exhaustive list of data-at-work publications. We're even unable to do this

Reply:
Let's add the deletions back in for now, and I'll merge. I'll revisit with the team after the holiday on how we want to handle removals. In most cases we wish we had more usage examples and tutorials, so this is a somewhat unique circumstance to be considering removing otherwise good examples.

Reply:
Ok, both items are added again, the paper now with the full author list (according to the style guide up to six author names are allowed).

Reply:
Much obliged, @sebastian-nagel ! I'll merge this right now.